drafts/2019-11-06-1900011.adoc

```diff
@@ -157,6 +160,95 @@ MetadataBuilder mdb = Metadata.builder();
 mdb = mdb.withName("metricName").withType(MetricType.COUNTER)
   .reusable(resolveIsReusable());
 ----
+[#JMO]
+== Jaeger for MicroProfile OpenTracing
+
+OpenLiberty can use Jaeger as a tracing backend for MicroProfile OpenTracing. 
+
+OpenLiberty has the support of MicroProfile OpenTracing. A sample tracer was provided at link:https://github.com/WASdev/sample.opentracing.zipkintracer[Opentracer] for using Zipkin as a tracing backend. With the introduction of jaeger-1.0, developers can also use Jaeger as a tracing backend.
+
+You will need to download the Jaeger client version 0.34.0 library and its dependencies from link:https://mvnrepository.com/artifact/io.jaegertracing/jaeger-client/0.34.0[Maven repository].
+
+In the server.xml 
+
+add `<feature>jaeger-1.0</feature> under <featureManager> section`
+
+[source, xml]
+----
+<library id="jaegerLib" apiTypeVisibility="api,ibm-api,spec,stable,third-party" >
+        <file name="<path>/jaegerLib_0.34/gson-2.8.2.jar" />
+        <file name="<path>/jaegerLib_0.34/jaeger-client-0.34.0.jar" />
+        <file name="<path>/jaegerLib_0.34/jaeger-core-0.34.0.jar" />
+        <file name="<path>/jaegerLib_0.34/jaeger-thrift-0.34.0.jar" />
+        <file name="<path>/jaegerLib_0.34/jaeger-tracerresolver-0.34.0.jar" />
+        <file name="<path>/jaegerLib_0.34/libthrift-0.12.0.jar" />
+        <file name="<path>/jaegerLib_0.34/slf4j-api-1.7.25.jar" />
+        <file name="<path>/jaegerLib_0.34/slf4j-jdk14-1.7.25.jar" />
+        <file name="<path>/jaegerLib_0.34/opentracing-util-0.31.0.jar" />
+        <file name="<path>/jaegerLib_0.34/opentracing-noop-0.31.0.jar" />
+    </library>
+----
+
+`- define your appplication:`
+
+[source, xml]
+
+ <webApplication location="yourapp.war" contextRoot="/yourapp">
+        <!-- enable visibility to third party apis -->
+        <classloader commonLibraryRef="jaegerLib"
+            apiTypeVisibility="api,ibm-api,spec,stable,third-party" />
+    </webApplication>
+
+You can find out more about about Jaeger settings set up using environment variables by looking at link:https://github.com/jaegertracing/jaeger-client-java/blob/10c641f8df6316f1eac4d5b1715513275bcd724e/jaeger-core/README.md[jaeger-client-java]
+
+** For the `JAEGER_PASSWORD`, the password can be encoded using the `securityUtility command`
+** Depending on Jaeger's sampling settings `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`, Jaeger may not report every spans created by the applications.
+
+[#LAC]
+== Liberty Annotation Caching
```

Comment:
Make heading match the link above.

Reply:
This section has too many line breaks. The sentences should be grouped in sensible paragraphs - not each in a separate paragraph.

"Annotation" should not be capitalised except if it's the start of a sentence.

Highlight properties and values with backticks (monospace) not quotations marks (eg third paragraph, or "--clean").

"java" must always be "Java" (unless it's a property name or something, of course)

Read through the whole section and fix typos like "cachedscan". Make sure there aren't any missing words etc.